website/docs/storybook.md

```diff
@@ -38,6 +38,36 @@ export const FormSignIn = {
 };
 ```
 
+## Interaction testing
+
+You can go further and use Interactors to write tests for your stories in Storybook with [interaction testing](https://storybook.js.org/blog/interaction-testing-with-storybook/) feature.
+
+To use it install `@storybook/addon-interactions` and `@interactors/with-storybook` storybook addons. Add them to your storybook config and enable interactions debugger feature:
```

Comment:
Is there anything particlar about adding them to the storybook config or is it very standard?